plugins/obs-ffmpeg/obs-ffmpeg-output.c

```diff
@@ -1117,6 +1133,7 @@ static int try_connect(struct ffmpeg_output *output)
 	const char *full_streamkey;
 
 	settings = obs_output_get_settings(output->output);
+	memset(&config, 0, sizeof(config));
```

Comment:
this actually fixed the crash i was experiencing as the video settings wasnt null and would later cause the application to crash when it tried to parse them